docs/rules/newline-after-var.md

```diff
@@ -23,107 +23,115 @@ This rule enforces a coding style where empty newlines are required or disallowe
 
 ## Options
 
-This rule takes one option, a string, which can be:
+This rule has a string option:
 
-* `"always"` enforces empty newlines after `var`, `let` or `const` (default)
-* `"never"` disallows empty newlines after `var`, `let` or `const`
+* `"always"` (default) requires an empty newline after `var`, `let`, or `const`
+* `"never"` disallows empty newlines after `var`, `let`, or `const`
 
-The following patterns are considered problems:
+### always
+
+Examples of **incorrect** code for this rule with the default `"always"` option:
 
 ```js
 /*eslint newline-after-var: ["error", "always"]*/
+/*eslint-env es6*/
 
 var greet = "hello,",
     name = "world";
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: ["error", "never"]*/
-/*eslint-env es6*/
 
 let greet = "hello,",
     name = "world";
 
```

Comment:
Is it possible that this sub-example is reversed?
* has empty line here in **incorrect** code
* does not have empty line in **correct** code (that is some kind of double negative :)